Gemfile

```diff
@@ -67,6 +67,7 @@ group :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
   gem 'selenium-webdriver'
+  gem 'shoulda-matchers'
```

Comment:
Would you mind removing this gem? I'd prefer fewer dependencies on the repo, and I'm not a huge fan of `shoulda-matchers`.